Assets/Scripts/Manager/GhostSwitchManager.cs

```diff
@@ -208,6 +211,7 @@ void possess (GameObject player, GameObject doll)
 
 		DollAudioManager.getInstance ().playGhostSwitchSound ();
 		updateCamera ();
+		updateAudio ();
```

Comment:
Shit sorry, didn't realize this was under the possess method, github made it look different